sfcc_cartridge/int_fast_sfra/cartridge/static/default/js/fastProduct.js

```diff
@@ -11,7 +11,31 @@ $(document).ready(function () {
         checkoutFastButton.removeClass('fast-checkout');
     }
 
+   // following code will exucute when click on quickview button which having class 'quickview'
+    $('body').on('click', '.quickview', function (e) {
+        e.preventDefault();
+        
+        setTimeout(function() {
+            initQuickviewFastBtn();
+        }, 1000);
+    });
+    
+    // following code will exucute when cart page product editProductModal open which render the quickview isml 
+    $('a[data-target="#editProductModal"]').click(function(e) {
```

Comment:
why is this not using a delegate like line#15

Reply:
**$('a[data-target="#editProductModal"]').click** code removed as its not needed